doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -503,6 +503,45 @@ set with::
 :Required: No
 :Default: 64K
 
+RocksDB Sharding
+==================
+
+BlueStore uses internally multiple types of key-value data, using RocksDB.
+Each data type in BlueStore is assigned unique prefix. Until Pacific all
+data was stored in single RocksDB data set: 'default'.
+Pacific has an ability to split all of BlueStore's key-value data
+into multiple RocksDB column families. When keys have similar access frequency,
+modification frequency and lifetime, we profit from better caching and more
+precise compaction. Need to compact some frequently changed keys does not force
+full compaction anymore.
+
+When OSD is deployed under Pacific, BlueStore sharding is applies.
+If Ceph is upgraded to Pacific from previous version, sharding if off.
+
+To apply sharding and effectively make BlueStore exactly the same as if
+deployed under Pacific, conversion can be done::
+
+  $ ceph-bluestore-tool --path <data path> --sharding="m(3) p(3,0-12) O(3,0-13)=block_cache={type=binned_lru} L P" reshard
```

Comment:
```suggestion
Internally BlueStore uses multiple types of key-value data,
stored in RocksDB.  Each data type in BlueStore is assigned a
unique prefix. Until Pacific all key-value data was stored in
single RocksDB column family: 'default'.  Since Pacific,
BlueStore can divide this data into multiple RocksDB column
families. When keys have similar access frequency, modification
frequency and lifetime, BlueStore benefits from better caching
and more precise compaction. This improves performance, and also
requires less disk space during compaction, since each column
family is smaller and can compact independent of others.

OSDs deployed in Pacific or later use RocksDB sharding by default.
If Ceph is upgraded to Pacific from a previous version, sharding is off.

To enable sharding and apply the Pacific defaults, stop an OSD and run::

    .. prompt:: bash #

      ceph-bluestore-tool --path <data path> --sharding="m(3) p(3,0-12) O(3,0-13)=block_cache={type=binned_lru} L P" reshard
```

Reworded a little and added a prompt for the command